getting_started/spm.md

```diff
@@ -17,7 +17,54 @@ tags: [spm]
 
-The developers of SPM have a formal collaboration with the developers of FieldTrip on many analysis issues. For example, SPM and FieldTrip share routines for converting data to MATLAB, forward modeling for M/EEG source reconstruction and the SPM12 distribution contains a version of FieldTrip so that you can combine FieldTrip and SPM functions in your custom scripts. SPM and FieldTrip complement each other well as SPM is geared toward very specific analysis tools as described above whereas FieldTrip is a more general repository of different methods that can be put together in flexible ways to perform a variety of analyses. In combined SPM12-FieldTrip the flexibility of FieldTrip can be complemented by the SPM GUI tools that allow rapid development of simple GUI's and also SPM12's new powerful batching system. Within this framework the power users in a lab can easily and rapidly develop specialized analysis tools with GUI that can then also be used by non-proficient MATLAB users. Some examples of such tools are available in the M/EEG toolbox distributed with SPM.
+The developers of SPM have a formal collaboration with the developers of FieldTrip on many analysis issues. For example, SPM and FieldTrip share routines for converting data to MATLAB, forward modeling for M/EEG source reconstruction and the SPM12 distribution contains a version of FieldTrip so that you can combine FieldTrip and SPM functions in your custom scripts. SPM and FieldTrip complement each other well as SPM is geared toward very specific analysis tools whereas FieldTrip is a more general repository of different methods that can be put together in flexible ways to perform a variety of analyses.
+
+[Reference paper](https://www.hindawi.com/journals/cin/2011/852961/), [courses](https://www.fil.ion.ucl.ac.uk/spm/course/video/) and [tutorials](https://jsheunis.github.io/2018-06-28-spm12-matlab-scripting-tutorial-1/) are also available
+
+
+## How does FieldTrip use SPM?
+
+As SPM's core expertise is mainly oriented to volumetric data processing (MRI, fMRI), FieldTrip integrates SPM functionalities to deal with two main operations:
+1. Creation of the geometries needed for accurate forward modelling (e.g. spm_create_vol, spm_segment)
+2. Spatial normalisation of volumetric images for efficient group statistics at the source level (e.g. spm_normalise, spm_deformations)
+The following figure shows main FieldTrip functions using SPM:
+
+![How FieldTrip uses SPM](/assets/img/getting_started/spm/FieldTrip_uses_SPM.png)
+
+
+## How does SPM use FieldTrip?
+
+SPM integrates some FieldTrip functions to perform operations on M/EEG data such as reading, format checking, plotting, preprocessing or frequency analysis. Specifically, SPM needs FieldTrip to create forward models (ft_compute_leadfield, ft_prepare_vol_sens).
+The following figure illustrates some SPM functions using FieldTrip:
+
+![How SPM uses FieldTrip](/assets/img/getting_started/spm/SPM_uses_FieldTrip.png)
```

Comment:
can you change this (and other images) according to https://www.fieldtriptoolbox.org/development/guideline/website/#images

Reply:
It should be ok now